SECURITY.md

```diff
@@ -9,6 +9,7 @@ If a security issue is discovered, please report it to security@riot-os.org.
 A response will be provided within one week.
 The issue will be tracked in the [security mailing list](mailto:security@riot-os.org).
 The original reporter will be included in the discussion of the issue.
+You can encrypt your report using gpg key id 44C6AE441172F88D3423E81F5F7964D0F4239033.
```

Comment:
Where can the public key be downloaded?

Reply:
FYI: I opened a PR to add the GPG key to the riot-os.org website. Maybe link to there as well, once it is merged.

Reply:
https://github.com/RIOT-OS/riot-os.org/pull/95

Reply:
Once merged, it will be at https://www.riot-os.org/assets/keys/security.gpg